src/Workspaces/CSharp/Portable/Formatting/Engine/Trivia/CSharpTriviaFormatter.cs

```diff
@@ -84,6 +84,25 @@ protected override LineColumnRule GetLineColumnRuleBetween(SyntaxTrivia trivia1,
                     return LineColumnRule.PreserveLinesWithGivenIndentation(lines: insertNewLine ? 1 : 0);
                 }
 
+                /* All the cases are Trivia inside ArrayInitializers
+                    case 1:  {
+                                // This is a comment after the brace
+                         <TriviaOfInterest>    2
+                    case 2:  ,
+                                // This is a comment after the comma of the ArrayInitializer
+                         <TriviaOfInterest>    2
+                    case 3:   <value>
+                            // This is a comment after one of the values for the ArrayInitializer
+                         <TriviaOfInterest>     ,
+                    case 4: // This is a comment after one of the values for the ArrayInitializer
+                         <TriviaOfInterest>     }
+                */
```

Comment:
why do this rather than using suppress operation? I thought array initializer is suppressed?